packages/kbn-build/src/utils/__snapshots__/link_project_executables.test.js.snap

```diff
@@ -42,3 +42,11 @@ Object {
   ],
 }
 `;
+
+exports[`bin script points to a file creates a symlink in the project node_modules/.bin directory: logs 1`] = `
+Array [
+  Array [
+    "[foo] bar -> ../bar/bin/bar.js",
```

Comment:
moved this to a snapshot, so it doesn't make noise in the jest output, also added a `stripAnsiSnapshotSerializer` to handle the output from `chalk` (otherwise there's a difference between running and watch-mode or not).